license

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Copy/pasting a quote from @jamiemagee:

> From some quick research, it seems like there is no issue with license compatibility. An excerpt from GNU[1](#user-content-fn-1-42db8b48175674d17d34aac6cdc0ae97):
> 
> > Being in the public domain is not a license; rather, it means the material is not copyrighted and no license is needed. Practically speaking, though, if a work is in the public domain, it might as well have an all-permissive non-copyleft free software license. **Public domain material is compatible with the GNU GPL**.
> 
> There is also a section on the Wikipedia page for the GPL about license compatibility[2](#user-content-fn-2-42db8b48175674d17d34aac6cdc0ae97) which uses the Free Software Foundation as a source to explicitly show the licenses as being compatible.
> 
> I understand that the GPL and AGPL are different licenses, but AGPLv2 is an extension of GPLv2, and is explicitly compatible.
> ## Footnotes
> 
>     1. https://www.gnu.org/licenses/license-list.en.html [↩](#user-content-fnref-1-42db8b48175674d17d34aac6cdc0ae97)
> 
>     2. https://en.wikipedia.org/wiki/GNU_General_Public_License#Compatibility_and_multi-licensing [↩](#user-content-fnref-2-42db8b48175674d17d34aac6cdc0ae97)

https://github.com/renovatebot/renovate/pull/15512#discussion_r885785507